src/docs/sdk/features.mdx

```diff
@@ -327,6 +330,71 @@ Automatically captured HTTP Client error events can be searchable and alertable
 
 As an example, see the [OkHTTP Client integration](https://github.com/getsentry/sentry-java/pull/2287) for Android.
 
+## GraphQL Client Integrations
+
+The GraphQL Client integrations match the [HTTP Client Integrations](/sdk/features/#http-client-integrations) with a few differences:
+
+The `failedRequestStatusCodes` parameter does not exist because GraphQL errors are not HTTP errors, sometimes the request is errored and the response is successful.
```

Comment:
```suggestion
The `failedRequestStatusCodes` parameter does not exist because GraphQL errors are not HTTP errors, so that the request can be errored even though the HTTP status code of the response is successful.
```